hr_expense_tax_adjustment/models/hr_expense_tax.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023-today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class HrExpenseTax(models.Model):
+
+    _name = "hr.expense.tax"
+    _description = "Taxes On Expense Lines"
+
+    expense_id = fields.Many2one(
+        "hr.expense", "Expense", required=True, ondelete="cascade", index=True
+    )
+    amount = fields.Float("Amount", required=True, digits=dp.get_precision("Account"))
```

Comment:
@lanto-razafindrabe No need to use dp.get_precision anymore 
we can use directly digits="Product Price" see [here](https://github.com/odoo/odoo/blob/16.0/addons/hr_expense/models/hr_expense.py#L65)